main.js

```diff
@@ -1,5 +1,7 @@
+let intervalID;
```

Comment:
Keep state related to the car within the car. Doing this makes it impossible for your app to work with more than one car.